db/seeds.rb

```diff
@@ -1,34 +1,43 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+# Read the seed data into memory. Do this before deleting data to be sure there isn't a problem
+seed_data = Dir.glob(Rails.root.join('db', 'seed', '**', '*.yml')).reduce({}) { |collection, file|
+  collection[File.basename(file, '.yml')] = YAML::load_file(Rails.root.join('db', 'seed', file))
+  collection
+}
 
-admin = User.create!(
-  name: "Rebecca Staples",
-  email: "rebecca@playtime.org",
-  admin: true
-)
+# Remove all seeded data.
+Wishlist.destroy_all
+User.destroy_all
+Item.destroy_all
+Pledge.destroy_all
+WishlistItem.destroy_all
+SiteManager.destroy_all
 
-user = User.create!(name: "Micah Bales", email: "micahbales@gmail.com")
+wishlists = Wishlist.create!(seed_data['wishlists'])
+users = User.create(seed_data['users'])
+items = Item.create(seed_data['items'])
 
-wishlist = Wishlist.create!(name: "DC General")
+# Associate users with wishlists in order
+site_managers = (0..5).collect { |index|
+  SiteManager.create!(user: users[index], wishlist: wishlists[index])
+}
 
-SiteManager.create!(user: user, wishlist: wishlist)
+# Randomly assign several items to a wishlist
+wishlist_items = wishlists.map { |wishlist|
+  5.times.map {
+    WishlistItem.create!(
+      quantity: 1,
+      wishlist: wishlist,
+      item: items.sample
```

Comment:
There's a small bug here: sampling allows for duplicates. We probably want something like `list.shuffle.take(n)` instead.

![duplicate sampled items](https://user-images.githubusercontent.com/814638/31872603-31bde950-b78b-11e7-97a8-13b4a02ed55f.png)

While we're here, could you invert this assignment (i.e. randomly assign an item to several wishlists)? That way, the wishlist size will vary a bit and all of the items will be used.